MekHQ/src/mekhq/campaign/stratcon/StratconCampaignState.java

```diff
@@ -169,6 +170,17 @@ public boolean isForceDeployedHere(int forceID) {
 
         return false;
     }
+    
+    /**
+     * Removes the scenario with the given campaign scenario ID from any tracks where it's present
+     */
+    public StratconScenario removeStratconScenario(int scenarioID) {
```

Comment:
Why is this returning anything, as the only return is null?

Reply:
I must have had other plans for this method at some point. Cleaned up.